src/posix.mak

```diff
@@ -34,9 +34,9 @@ LDFLAGS=-lm -lstdc++ -lpthread
 #ifeq (osx,$(OS))
 #	HOST_CC=clang++
 #else
-	HOST_CC=g++
+	HOST_CC=gcc
 #endif
-CC=$(HOST_CC) $(MODEL_FLAG)
+CC=$(HOST_CC) -x c++ $(MODEL_FLAG)
```

Comment:
This will not work.  The offending problem is appending the `$(MODEL_FLAG)`
Proof:  Run this: